src/test/resources/emails/summaryOfFeedbackSessionsOfCourseEmailForStudent.html

```diff
@@ -0,0 +1,56 @@
+Hello student1 In Course1</td></div>'",
+
+<p/>
+An instructor has updated your email address in the course [Course name: Typical Course 1 with 2 Evals] [Course ID: idOfTypicalCourse1].
+Given below are the relevant links for the course, updated to work with your new email address student1InCourse1@gmail.tmt.
+
+<p/>
+
+
+<br>
+Below are the feedback sessions of the course.
+<br>
+
+<p/>
+<br>
+<br>&nbsp;&nbsp; <strong>Feedback Session Name:</strong> Closed Session
+<br>&nbsp;&nbsp; <strong>Deadline:</strong> Sat, 01 Jun 2013, 11:59 PM {Passed}
+<br>
+To submit feedback for the above session, please go to this Web address: <a href="{Feedback session is closed}">{Feedback session is closed}</a>
+<br>
+<br>
+To view the responses for this session, please go to this Web address: <a href="{Feedback session is unpublished}">{Feedback session is unpublished}</a>
+<br>
+<br><br>
+<br>&nbsp;&nbsp; <strong>Feedback Session Name:</strong> First feedback session
+<br>&nbsp;&nbsp; <strong>Deadline:</strong> Fri, 30 Apr 2027, 11:59 PM
+<br>
+To submit feedback for the above session, please go to this Web address: <a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student1InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student1InCourse1%40gmail.tmt</a>
+<br>
+<br>
+To view the responses for this session, please go to this Web address: <a href="{Feedback session is unpublished}">{Feedback session is unpublished}</a>
```

Comment:
Is it possible to have a not-yet-opened session and a published session as well so that we can have complete branch coverage?